crypto/asn1/asn1_lib.c

```diff
@@ -414,35 +414,36 @@ unsigned char *ASN1_STRING_data(ASN1_STRING *x)
 #endif
 
 char *ossl_sk_ASN1_UTF8STRING2text(STACK_OF(ASN1_UTF8STRING) *text,
-                                   const char *sep,
-                                   size_t max_len /* excluding NUL terminator */)
+                                   const char *sep, size_t max_len)
+/* max_len excludes NUL terminator and may be 0 to indicate no restriction */
```

Comment:
put this at the top of the function, and consider writing it as `|max_len|` as is common for indicating parameters.

Reply:
Good suggestions; done.